tests/composite_pk/test_update.py

```diff
@@ -175,3 +175,25 @@ def test_cant_update_to_unsaved_object(self):
 
         with self.assertRaisesMessage(ValueError, msg):
             Comment.objects.update(user=User())
+
+    def test_bulk_update_max_bulk_batch_size(self):
```

Comment:
This is failing on Oracle for me locally with:
```
django.db.utils.DatabaseError: ORA-00907: missing right parenthesis
```

Either I have done the number of fields wrong or perhaps need to look into `max_query_params` for Oracle (see ticket-28596)